resources/events.ts

```diff
@@ -13,11 +13,15 @@ export class Events extends BaseResource {
     }
 
     public async list(params?: EventListParams): Promise<UnitResponse<UnitEvent[]>> {
-        const parameters = {
+        const parameters: Record<string, unknown> = {
             "page[limit]": (params?.limit ? params?.limit : 100),
             "page[offset]": (params?.offset ? params?.offset : 0)
         }
 
+        if(params?.filter?.type){
+            parameters["filter[type]"] = params.filter.type
```

Comment:
have you tested this function with an array of strings and verified that the default url encoding of arrays is accepted by Unit's server? (see example [here](https://stackoverflow.com/questions/49944387/how-to-correctly-use-axios-params-with-arrays) of adjusting axios encoding of array URL params)

Reply:
I've tested the encoding as part of the smoke tests in the etl.